scripts/local-airflow.sh

```diff
@@ -16,5 +17,11 @@ ln -s $PWD/$target_dag $AIRFLOW_HOME/dags/
 
 cd $AIRFLOW_HOME
 
+# Disable auth login
+echo "AUTH_ROLE_PUBLIC = 'Admin'" > webserver_config.py
+
 export AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES_REGEXP='.*'
+export AIRFLOW__CORE__LOAD_EXAMPLES=False
```

Comment:
Also a good find!

Reply:
Probably we could add this file as comment to this script? https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html